src/naip_cnn/inference.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+import json
+
+import tensorflow as tf
+
+
+def parse_imagery_tfrecord(
+    serialized_example: bytes,
+    bands: tuple[str] = ("R", "G", "B", "N"),
```

Comment:
Does it make sense to import `BANDS` from `.config` and use here?

Reply:
Totally, I made the change!